presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -58,6 +58,7 @@ services:
     environment:
       - PRESTO_JDBC_DRIVER_CLASS
       - TESTS_HIVE_VERSION_MAJOR
+      - TESTS_HIVE_IS_HIVE_VERSION_BEFORE_12
```

Comment:
can we somehow have a single variable for hive version? Like `TESTS_HIVE_VERSION` that would contain major and minor part of version. Then we could write some logic what should be executed on which Hive version.

I am afraid that at some point we could end up with `TESTS_HIVE_IS_HIVE_VERSION_AFTER_12_AND_BEFORE_20` etc.

Reply:
i consciously decided not to.
I don't want any version parsing logic.

Reply:
> i consciously decided not to.

Why is so?

> I don't want any version parsing logic.

You already have such. That validates `TESTS_HIVE_IS_HIVE_VERSION_BEFORE_12`.

Maybe `TESTS_HIVE_WITH_SUPPORT_FOR_DATA_COLUMNS_STATISTICS`?

Reply:
I will replace `TESTS_HIVE_IS_HIVE_VERSION_BEFORE_12` with `TESTS_HIVE_VERSION_MINOR`, ok?
ie still two variables (and no version parsing logic).

Reply:
Done